tests/regress/hook_add_crash.py

```diff
@@ -1,21 +1,15 @@
-#!/usr/bin/env python
-
-"""https://github.com/unicorn-engine/unicorn/issues/165"""
+""" https://github.com/unicorn-engine/unicorn/issues/165 """
 
 import regress
-
 from unicorn import *
 
 
-def hook_mem_read_unmapped(mu, access, address, size, value, user_data):
-    pass
-
 class TestHook(regress.RegressTest):
     def test_excessive_hooks(self):
         mu = Uc(UC_ARCH_X86, UC_MODE_32)
 
         for _ in range(1337):
-            mu.hook_add(UC_HOOK_MEM_READ_UNMAPPED, hook_mem_read_unmapped)
+            mu.hook_add(UC_HOOK_MEM_READ_UNMAPPED, lambda *args, **kwargs: None)
```

Comment:
Python linters strongly advise against `lambda` functions, and I am not sure there is a justification to deviate from this recommendation here. Also, this is not only a style change - but also functional: while the older implementation re-uses the same function, the new one creates a new `lambda` function for each iteration.